cldf-metadata.json

```diff
@@ -0,0 +1,298 @@
+                        }
+                    },
+                    {
+                        "name": "VALUE",
+                        "datatype": {
+                            "base": "string"
+                        }
+                    }
+                ],
+                "primaryKey": [
+                    "ID"
+                ]
+            }
+        },
+        {
+            "url": "data/graphemes.tsv",
+            "tableSchema": {
+                "columns": [
+                    {
+                        "name": "PK",
+                        "datatype": {
+                            "base": "integer"
+                        }
+                    },
+                    {
+                        "name": "GRAPHEME",
+                        "dc:description": "Grapheme used in a particular transcription to denote a sound",
+                        "datatype": {
+                            "base": "string"
+                        }
+                    },
+                    {
+                        "name": "NAME",
+                        "dc:description": "The ordered concatenation of feature values of the denoted sound",
+                        "datatype": {
+                            "base": "string"
+                        }
+                    },
+                    {
+                        "name": "BIPA",
+                        "dc:description": "The grapheme for the denoted sound in the Broad IPA transcription system",
+                        "datatype": {
+                            "base": "string"
+                        }
+                    },
+                    {
+                        "name": "DATASET",
+                        "dc:description": "Links to the source of this grapheme"
+                    },
+                    {
+                        "name": "FREQUENCY",
+                        "datatype": {
+                            "base": "integer"
+                        }
+                    },
+                    {
+                        "name": "URL",
+                        "dc:description": "URL of the grapheme in its source online database",
+                        "datatype": {
+                            "base": "anyURI"
+                        }
+                    },
+                    {
+                        "name": "IMAGE",
+                        "dc:description": "Image of the typeset grapheme.",
+                        "valueUrl": "http://web.uvic.ca/ling/resources/ipa/charts/IPAlab/images/{IMAGE}",
+                        "datatype": {
+                            "base": "anyURI"
+                        }
+                    },
+                    {
+                        "name": "SOUND",
+                        "dc:description": "Audio recording of the sound being pronounced.",
+                        "valueUrl": "http://web.uvic.ca/ling/resources/ipa/charts/IPAlab/IPAsounds/{SOUND}",
```

Comment:
@LinguList If this is the location you wanted to link to for IPA sound files, the `IPASounds/` prefix for the file names in the data should be dropped.

Reply:
Oh, good you caught this. Not sure if they changed their URL, or I did not pay attention...